lib/src/component2_suggestors/component2_constants.dart

```diff
@@ -35,3 +35,28 @@ String getDeperecationMessage(String methodName) {
   ///
   $revertInstructions''';
 }
+
+final componentWillMountMessage = '''
+  /// FIXME: Method has been updated from `componentWillMount` to `componentDidMount`. Please check if super call should be added, updated, or removed.''';
+
+final deprecatedLifecycleMethods = [
```

Comment:
#nit this could be `const` instead, which for Lists, has the advantage of also making the list immutable:
```suggestion
const deprecatedLifecycleMethods = [
```
This helps ensure this global variable isn't unintentionally mutated.